finding_CommonID.jl

```diff
@@ -0,0 +1,31 @@
+using CSV, DataFrames
+
+### Finding matches to "cleared" recovered Malawi samples (Guiliherme) 
+
+#Load the csv with all the Malawi samples that we have in lab, "tenative" samples. 
+tentative_samples_list = CSV.read("", DataFrame) #DataFrame is the type that it produces
+
+#Load the csvs with all the recovered Malawi samples that are 'cleared' 
+ID1 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T1_10.03.2024.xlsx")) #dropmissing 
+ID2 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T2_10.03.2024.xlsx"))
+ID3 = dropmissing(CSV.read("KhulaMW_BiospecimenTemplate_T3_10.03.2024.xlsx"))
```

Comment:
Ideally, as the project gets more mature, we will transition from those filenames to something different. Usually, we do not share Excel or CSV files containing subject metadata on github. Let's talk about that in the future =]

Reply:
👍